src/test/groovy/com/github/mkopylec/charon/specification/ProxyingRequestSpec.groovy

```diff
@@ -145,6 +145,7 @@ abstract class ProxyingRequestSpec extends BasicSpec {
 
         then:
         assertThat(response)
-                .hasStatus(MOVED_PERMANENTLY)
+                .hasStatus(OK)
+                .bodyContains("<link rel=\"canonical\" href=\"https://github.com/\"")
```

Comment:
See comment in the body of this PR -- the test was actually failing before, i.e. the old assertion was wrong.
After fixing the `Host` header issue, the request now works, so I have fixed the assertion.